experimental/Matrix/matrix.jl

```diff
@@ -0,0 +1,12 @@
+module MatrixGroupExperimental
+
+using GAP
+using Oscar
+  
+function __init__()
+    GAP.Globals.Reread(GAP.GapObj(joinpath(Oscar.oscardir, "experimental", "Matrix", "matrix.g")))
```

Comment:
OK, assuming this works, the next thing would be to add some code (GAP or Julia) which instantiates `IsJuliaMatrixRep` and tests it. Perhaps something like this for starters:
```julia
_wrap_for_gap(m::MatrixElem) = GAP.Globals.MakeJuliaMatrixRep(m)

m = matrix(ZZ, [0 1 ; -1 0])
gapM = _wrap_for_gap(m)
G = GAP.Globals.Group(gapM)
@test GAP.Globals.Size(G) == 4
```

Reply:
I tried to add some tests to the test folder but somehow the code is not executed. Do you know why?